one_fm/grd/doctype/work_permit/work_permit.json

```diff
@@ -345,11 +344,11 @@
    "fieldtype": "Column Break"
   },
   {
+   "fetch_from": "employee.pam_file",
    "fieldname": "pam_file",
-   "fieldtype": "Link",
```

Comment:
PAM File is a link and we need to set PAM File Number in the Work Permit

Reply:
Concerning this, i have dropped a picture in your goggle chat